### ISSUE TITLE
Unable to install with pyton 3.10

### ISSUE THREAD

**Issue by Enrikaa** [mapado/haversine#47]
Hello!

Thanks for creating this module. It helped a lot to make the distance calculations. Unfortunately, we can't use it in latest Python version.

Right now this lib only supports Python max 3.9 version. So there are dependency conflicts when we want to use latest Python==3.10 and haversine==2.5.1 library.

**Comment by jdeniau** [comment]
Hi,

I'm not a python developer so I don't know the package installation details.
In `setup.py`, it does say :
```
    python_requires='>=3.5',
```

So I think it should work.
There is a list of classifier later that include all python version between 3.5 and 3.9 though.

Do you know how to fix this? If so, can you open a PR?

Thanks

**Comment by merschformann** [comment]
I just installed and tested _haversine_ in a clean docker with python `3.10.5`. All seems to be working fine.

Could you provide more details about what is going wrong?

In order to reproduce my test, do the following:

1. Start interactive python container
    
    ```bash
    docker run --rm -it python:3.10 /bin/bash
    ```

2. Inside the container run

    ```bash
    pip install haversine
    python << EOM
    from haversine import haversine
    lyon = (45.7597, 4.8422)
    paris = (48.8567, 2.3508)
    print(haversine(lyon, paris))
    EOM
    ```

3. You should get the output (among the pip install logs)

    ```bash
    392.2172595594006
    ```

**Comment by jdeniau** [comment]
@Enrikaa can you confirm that your issue is still open? and if so, can you provide a reproducible case?

Otherwise I will close this next week.

Thanks

**Comment by Enrikaa** [comment]
Hey, 

Thank you so much for the answer. I was upgrading project to 3.10 Python and looked that PyPi wrote that Haversine supports only >3.9. But as I investigated more - sometimes PyPi may not write the exact Python version support history. So I did the same thing as you mentioned above and everything is working. This issue can definitely be closed.  

Once again, thank you very much for the Haversine, it is very useful for us.
